docs/reporting/overview.md

```diff
@@ -0,0 +1,56 @@
+# Overview
+
+Reporting API allows you to query all transactions taking place at any of your MobilePay payment locations (payment points).
+
+The main objective is to make it easier for you to do reconciliation on your side. We at MobilePay will provide details of transactions to help you with that, with the products that MobilePay offers. This will be done by summarizing payment, fees, refunds. These are summarized in further detail below.
+
+:::info
+Reporting API let's you fetch transactions from **all products** offered by MobilePay:
+
+- App payments
+- Subscriptions
+- Invoice
+- MyShop
+- PoS
+
+:::
+
+
+## Migration from transaction reporting API
+
+Reporting API is a newer version of Transaction Reporting API, more about the old API you can find [here](https://mobilepaydev.github.io/MobilePay-TransactionReporting-API/).
+
+Also, you can use this guide on [how to migrate](../reporting/migrate-from-v1v2) from the old API.
+
+## How does it work?
+
+A settlement payout includes all your sales transactions from the previous day, net refunds and fees.
+By using the API, you get a full list of all sales transactions and corresponding fees, totalling the settlement on your bank account.
+You can reconcile your accounts with a high degree of data and transparency, moving your business towards always having up-to-date financial overviews.
+
+:::note
+If you chose to have fees invoiced, then fees will not be deducted directly by MobilePay, and instead you will need to reconcile against the invoices you receive.
+**Invoiced fees are not visible in the API.**
+:::
+
+## General notes
+
+:::important
+
+This document does not represent all of the technical information about Reporting API.
+This information could be found [here](../../api/reporting).
+
+:::
+
+:::note Payment Point Id
+
+When using the Reporting API, you will be introduced to the term `paymentPointId` - uuid assigned to every payment point.
+MobilePay has different API products and each product has their own payment points. Payment point might named differently across products.
```

Comment:
Don't get the point of this sentence?

Reply:
Idk, pretty clear to me. Can you elaborate?

Reply:
Missing "be": "might **be** named differently"

Reply:
might be

Reply:
we talked on teams. Changed to:

> `paymentPointId` might be named differently across products: